src/storage/src/hummock/state_store.rs

```diff
@@ -395,7 +395,13 @@ impl StateStore for HummockStorage {
             // compaction_group_id in read/write path.
             let size = self
                 .local_version_manager
-                .write_shared_buffer(epoch, compaction_group_id, kv_pairs, false)
+                .write_shared_buffer(
+                    epoch,
+                    compaction_group_id,
+                    kv_pairs,
+                    false,
+                    write_options.table_id.into(),
```

Comment:
Why we need an `into()` here? Will we globally unify the type table_id in our system to something like `TableId`?

Reply:
we need to unify the type of `table_id ` in our system in other PR